src/ReleasableCamera.cpp

```diff
@@ -14,7 +14,7 @@ namespace ZividPython
 {
     void wrapClass(pybind11::class_<ReleasableCamera> pyClass)
     {
-        pyClass.def(py::init<>())
+        pyClass.def(py::init())
             .def(py::self == py::self) // NOLINT
```

Comment:
Im am not sure if there are any comparison operators in the wrapped C++ code, but you should check if it is and add operator for those?

Side note: Also are we sure all of these operators are actually in the python layer as well. Should check that, but fixing that is an other issue.

Reply:
We are not using any of the equality operators in the `zivid` module, will create an issue for it